src/Abp.ZeroCore.EntityFramework/Abp.ZeroCore.EntityFramework.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <Import Project="..\..\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Target `netstandard2.0` — same for:
- line 16: `<None Include="bin\Release\net461\Abp.ZeroCore.EntityFramework.pdb">`
- line 17: `<PackagePath>lib/net461/</PackagePath>`